catalog/spatial/csw/spatial-csw-source-common/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/reader/TestGetRecordsMessageBodyReader.java

```diff
@@ -118,9 +114,14 @@ public void testConfigurationArguments() throws Exception {
 
     @Test
     public void testFullThread() throws Exception {
-        CswTransformProvider provider = new CswTransformProvider(null, mockInputManager);
-
-        when(mockInputManager.getTransformerBySchema(anyString())).thenReturn(new CswRecordConverter());
+        List<Metacard> inputMetacards = new ArrayList<>();
```

Comment:
Is there a reason you choose to mock out a new provider instead of using the CswTransformProvider as before?